File: android-kit-base/src/test/java/com/mparticle/kits/CommerceEventUtilsTest.java
package com.mparticle.kits;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotNull;

public class CommerceEventUtilsTest {

    @Test
    public void testNullProductExpansion() throws Exception {
        assertNotNull(CommerceEventUtils.expand(null));
        assertEquals(0, CommerceEventUtils.expand(null).size());
    }
}

File: android-kit-base/src/test/kotlin/com/mparticle/kits/CommerceEventUtilsTest.kt
package com.mparticle.kits

import com.mparticle.MParticle.EventType
import com.mparticle.commerce.CommerceEvent
import com.mparticle.commerce.Product
import org.junit.Assert
import org.junit.Test

class CommerceEventUtilsTest {
    @Test
    @Throws(Exception::class)
    fun testNullProductExpansion() {
        Assert.assertNotNull(CommerceEventUtils.expand(null))
        Assert.assertEquals(0, CommerceEventUtils.expand(null).size.toLong())
    }
}
